### PR TITLE
Fixes all broken links and enforces no broken

### PR DESCRIPTION
**Pull request by supertopher**
## What
we had some broken links

## How
fix broken links
enforce no broken links compilation

### CODE REVIEW

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # Table of contents
 
-- [Introduction](../README.md)
+- [Introduction](./readme.md)
 - [Airbyte Cloud QuickStart](cloud/getting-started-with-airbyte-cloud.md)
   - [Core Concepts](cloud/core-concepts.md)
   - [Managing Airbyte Cloud](cloud/managing-airbyte-cloud.md)
-- [Airbyte Open Source Quickstart](quickstart/README.md)
+- [Airbyte Open Source Quickstart](quickstart/deploy-airbyte.md)
```
Comment (supertopher) [inline docs/SUMMARY.md:7]: this readme and some others didn't exist.  Links to folders without a readme like `/quickstart/` don't work in docusaurus.  I pointed this and other top level links to a markdown file in the folder

File: docusaurus/docusaurus.config.js [hunk 2]
```diff
@@ -13,8 +13,8 @@ const config = {
     // Assumed relative path.  If you are using airbytehq.github.io use /
     // anything else should match the repo name
     baseUrl: '/',
-    onBrokenLinks: 'warn',
-    onBrokenMarkdownLinks: 'warn',
+    onBrokenLinks: 'throw',
```
Comment (supertopher) [inline docusaurus/docusaurus.config.js:16]: these lines make it so we can't deploy with broken links unless you specify a command to ignore the warnings

File: docs/archive/examples/slack-history.md [hunk 3]
```diff
@@ -102,9 +102,8 @@ Download \(or copy and paste\) this [html file](https://github.com/airbytehq/air
 
 ## 5. "Productionizing" Saving Slack History
 
-You can find instructions for how to host Airbyte on various cloud platforms [here](../../deploying-airbyte).
+You can find instructions for how to host Airbyte on various cloud platforms [here](../../deploying-airbyte/local-deployment.md).
```
Comment (git-phu) [inline docs/archive/examples/slack-history.md:105]: this one too feels like it wouldn't make sense to link to a specific deployment option

File: docs/archive/faq/getting-started.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 ## **What do I need to get started using Airbyte?**
 
-You can deploy Airbyte in several ways, as [documented here](../../deploying-airbyte/). Airbyte will then help you replicate data between a source and a destination. Airbyte offers pre-built connectors for both, you can see their list [here](../../project-overview/changelog/connectors.md). If you don’t see the connector you need, you can [build your connector yourself](../../connector-development) and benefit from Airbyte’s optional scheduling, orchestration and monitoring modules.
+You can deploy Airbyte in several ways, as [documented here](../../deploying-airbyte/local-deployment.md). Airbyte will then help you replicate data between a source and a destination. Airbyte offers pre-built connectors for both, you can see their list [here](../../project-overview/changelog/connectors.md). If you don’t see the connector you need, you can [build your connector yourself](../../connector-development) and benefit from Airbyte’s optional scheduling, orchestration and monitoring modules.
```
Comment (git-phu) [inline docs/archive/faq/getting-started.md:5]: ditto

File: docs/operator-guides/configuring-sync-notifications.md [hunk 5]
```diff
@@ -36,7 +36,7 @@ Click `Copy.`
 
 **Add the webhook to Airbyte.**
 
-Assuming you have a [running instance of Airbyte](../deploying-airbyte/README.md), we can navigate to the UI. Click on Settings and then click on `Notifications`.
+Assuming you have a [running instance of Airbyte](../deploying-airbyte/local-deployment.md), we can navigate to the UI. Click on Settings and then click on `Notifications`.
```
Comment (git-phu) [inline docs/operator-guides/configuring-sync-notifications.md:39]: ditto

File: docs/understanding-airbyte/glossary.md [hunk 6]
```diff
@@ -18,7 +18,7 @@ Stands for **E**xtract, **T**ransform, and **L**oad and **E**xtract, **L**oad, a
 
 **Extract**: Retrieve data from a [source](../integrations/sources/), which can be an application, database, anything really.
 
-**Load**: Move data to your [destination](../integrations/destinations/).
+**Load**: Move data to your [destination](../integrations/destinations/amazon-sqs.md).
```
Comment (git-phu) [inline docs/understanding-airbyte/glossary.md:21]: ditto

**Review by git-phu — COMMENTED** [review]
cool, great idea!

I do think some of the proposed links in this PR would be a bit confusing though, so I feel like we can either add new pages in this PR, or we have to take note of all the placeholder pages we used in this PR and in a subsequent PR update all these links to point to the pages/content we actually want.

File: docs/SUMMARY.md [hunk 7]
```diff
@@ -1,15 +1,15 @@
...
   - [Deploy Airbyte](quickstart/deploy-airbyte.md)
   - [Add a Source](quickstart/add-a-source.md)
   - [Add a Destination](quickstart/add-a-destination.md)
   - [Set up a Connection](quickstart/set-up-a-connection.md)
-- [Deploying Airbyte Open Source](deploying-airbyte/README.md)
+- [Deploying Airbyte Open Source](deploying-airbyte/local-deployment.md)
```
[the remark this answers is not in the document]
Comment (supertopher) [reply docs/SUMMARY.md:12]: agreed.  I knew this one felt harry and I think I punted when I should have just made a landing page

File: docs/SUMMARY.md [hunk 8]
```diff
@@ -182,7 +182,7 @@
     - [Zoho CRM](integrations/sources/zoho-crm.md)
     - [Zoom](integrations/sources/zoom.md)
     - [Zuora](integrations/sources/zuora.md)
-  - [Destinations](integrations/destinations/README.md)
+  - [Destinations](integrations/destinations/amazon-sqs.md)
```
[the remark this answers is not in the document]
Comment (supertopher) [reply docs/SUMMARY.md:185]: My thought was to force open the sidebar to show all the files by selecting the first member of the list, but I agree your solution is more better

**Review by supertopher — COMMENTED** [review]
thanks for the Review Peter, insightful as always

I think I was trying to get this out the door to fix links and cut a few corners that you have convinced me are worth tackling.  I'll make a new PR and ping you after that

File: docs/SUMMARY.md (hunk 1 above)
Comment (supertopher) [reply docs/SUMMARY.md:7]: @git-phu I haven't looked at the documentation... though this conversation is probably about to change that.  From testing, docusaurus doesn't appear to understand `/folders/` I believe this is a quirk of the manual sidebar we are using.  As stated elsewhere I think I'll just make a quick README in the places that didn't have one

File: docs/SUMMARY.md [hunk 9]
```diff
@@ -1,15 +1,15 @@
 # Table of contents
 
-- [Introduction](../README.md)
+- [Introduction](./readme.md)
```
Comment (supertopher) [inline docs/SUMMARY.md:3]: @Amruta-Ranade has pointed out this whole file is an artifact of the gitbook days, so imma murder it

**Comment by supertopher** [comment]
Following @tuliren 's advice for using GitHub as our `imugr` clone :) 

image

File: docs/deploying-airbyte/README.md [hunk 10]
```diff
@@ -0,0 +1,14 @@
+# Deploy Airbyte where you want to
```
Comment (supertopher) [inline docs/deploying-airbyte/README.md:1]: following @git-phu 's advice I made a ToC

File: docs/deploying-airbyte/on-aws-ecs.md [hunk 11]
```diff
@@ -6,5 +6,7 @@ We do not currently support deployment on ECS.
 
 :::
 
-The current iteration is not compatible with ECS. Airbyte currently relies on docker containers being able to create other docker containers. ECS does not permit containers to do this. We will be revising this strategy soon, so that we can be compatible with ECS and other container services.
-
+The current iteration is not compatible with ECS.
```
Comment (supertopher) [inline docs/deploying-airbyte/on-aws-ecs.md:9]: markdown doesn't care about single carriage returns and I think this read easier for people in shell and text editors

**Review by supertopher — COMMENTED** [review]
took basically all of Peter's advice

**Review by tuliren — APPROVED** [review]

**Review by git-phu — COMMENTED** [review]
Thanks for making all the changes!

**Pull request merged**